s3/proto/s3.proto

```diff
@@ -135,7 +154,7 @@ message Bucket {
 	WebsiteConfiguration websiteConfiguration = 11;
 	CORSConfiguration cors = 12;
 	ReplicationConfiguration replicationConfiguration = 13;
-	repeated LifecycleRole lifecycleConfiguration = 14;
+	repeated LifecycleRule LifecycleConfiguration = 14;

```

Comment:
first letter of LifecycleConfiguration is lowercase

Reply:
s/LifecycleConfiguration/lifecycleConfiguration

Reply:
Why replace Action with ExpirationAction and TransitionAction ? Could you give some explanation here?